scripts/example-analysis.jl

```diff
@@ -0,0 +1,29 @@
+# First, read in the output energies and iterations as a DataFrame
+df = read_output("scripts/LJ_111_p08_output_df.csv")
+
+# Calculate the ω factors
+ωi = ωᵢ(df.iter, 640)
+
+# Shift the energies to be positive
```

Comment:
Instead of "positive," it'd be more precise to say "greater than or equal to zero."

Reply:
Cool!